RealmSwift/Object.swift

```diff
@@ -212,6 +212,38 @@ extension Object: RealmCollectionValue {
      transactions it will be called at some point in the future after the write
      transaction is committed.
 
+     If no key paths are given, the block will be executed on any insertion,
+     modification, or deletion for all object properties and nested linked
+     properties (up to a depth of 4). If a key path or key paths are provided,
```

Comment:
Same as for `Aliases`.

Reply:
Resolved in [b8e9ee0](https://github.com/realm/realm-cocoa/pull/7249/commits/b8e9ee02afed35f06f77bd688886ad6b7aea627d)